### PR TITLE
Feature/add fusion auth

### PR DESCRIPTION
**Pull request by Mmishaaa**

### CODE REVIEW

File: Tinder.BLL/Extensions/Extension.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Tinder.BLL.Extensions
+{
+    public static class Extension
+    {
+        public static int Age(this DateTime dateTime)
```
Comment (moltenessence) [inline Tinder.BLL/Extensions/Extension.cs]: naming?

File: Tinder.BLL/Models/User.cs [hunk 2]
```diff
@@ -5,11 +5,12 @@ namespace Tinder.BLL.Models
     public class User
     {
         public Guid Id { get; set; }
+        public Guid FusionId { get; set; }
```
Comment (moltenessence) [inline Tinder.BLL/Models/User.cs]: naming is not so clear

Comment (Mmishaaa) [reply Tinder.BLL/Models/User.cs]: which naming is clear
FusionAuthId?

File: Tinder.BLL/Extensions/Extension.cs (hunk 1 above)
Comment (Mmishaaa) [reply Tinder.BLL/Extensions/Extension.cs]: is calculateAge will be clear?

**Review by VadimZharikov — APPROVED** [review]

**Review by pavelGagarin0870 — APPROVED** [review]

**Pull request merged**